java/streaming/src/test/resources/ray.conf

```diff
@@ -0,0 +1,7 @@
+ray {
+  home = "./ray"
+  //run-mode = CLUSTER
+  run-mode = SINGLE_PROCESS
+  resources: "CPU:4,GPU:0"
```

Comment:
Use `resources=` to make the style constant and the same to `ava/streaming/src/main/resources/ray.conf`